EMS/core-bundle/src/Controller/ContentManagement/EnvironmentController.php

```diff
@@ -361,6 +372,8 @@ public function removeAction(int $id): Response
             }
         }
 
+        $this->environmentRepository->updateOrderKeysAfterDelete($environment->getOrderKey());
+
```

Comment:
I don't get it

Reply:
If i delete environment number 3, without this the orderKeys are: 1 2 4 5 6 ... And with it : 1 2 3 4...

Reply:
Why is it a problem? Who cares that there is missing counters in the order list?